tests/test_docs.py

```diff
@@ -121,8 +121,15 @@ def extract_code_blocks(self, source: str, filename: str):
         parser.parse(source, document)
 
         lines = source.split('\n')
+        lint_on = True
         for lineno, line in enumerate(lines, 1):
-            if len(line) > self.MAX_LINE_LEN:
+
+            if line.startswith('.. lint-off'):
+                lint_on = False
+            elif line.startswith('.. lint-on'):
+                lint_on = True
```

Comment:
Can you also add basic pairing validation? Basically lint-off should always be followed by lint-on before EOF. That would prevent disabling lint accidentally after merge conflicts.

Reply:
Done